eisbuk-admin/src/components/layout/AttendanceSheet.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
```

Comment:
For both files: This shouldn't be inside layout folder. For now you can put them in `atoms`, as we're mostly using that folder for new components...we will reorganize the folder structure later. 
The `layout` is for some layout components shared throughout the pages: admin bar, some shared headers (if we had any). Visually, you should think of layout as a frame for the content: so that the pages change the content inside the frame, while the frame stays the same (menu, logo, nav, etc.)

Reply:
And please put the `AttendanceSheet` in it's own folder, with `index.ts` and stories, like we're doing with all components.